general/1_terminal/Daniil-Zymenok/task4.sh

```diff
@@ -0,0 +1,3 @@
+#!bin/bash
+
+jq '.messages[]' result.json | jq '.text'
```

Comment:
You can chain together `jq` filters by using `jq filterOne | filterTwo`, no need to chain together 2 `jq` processes.